From 5d1ac2e0d5f61913aad62dadb65a7fea6f1b93ae Mon Sep 17 00:00:00 2001
From: Shahbaz Youssefi <syoussefi@chromium.org>
Date: Tue, 17 Jan 2023 10:21:01 -0500
Subject: [PATCH] Add missing include to ShaderVars.h

Patch by mcatanza@redhat.com

Bug: angleproject:7963
Change-Id: I1466ab227ae4e8b5703bd68bafbf12c31d441c2e
Reviewed-on: https://chromium-review.googlesource.com/c/angle/angle/+/4174172
Reviewed-by: Amirali Abdolrashidi <abdolrashidi@google.com>
Auto-Submit: Shahbaz Youssefi <syoussefi@chromium.org>
Commit-Queue: Shahbaz Youssefi <syoussefi@chromium.org>
---
 include/GLSLANG/ShaderLang.h | 2 +-
 include/GLSLANG/ShaderVars.h | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/GLSLANG/ShaderVars.h b/include/GLSLANG/ShaderVars.h
index 94cb93e01fc..ec7bda372f3 100644
--- a/src/third_party/angle/include/GLSLANG/ShaderVars.h
+++ b/src/third_party/angle/include/GLSLANG/ShaderVars.h
@@ -12,6 +12,7 @@
 
 #include <algorithm>
 #include <array>
+#include <cstdint>
 #include <string>
 #include <vector>
 
